### PR TITLE
Stub release for 0.2.0

### PR DESCRIPTION
**Pull request by alex-ioja-yang-cko**
Automated release prepared. Lets get ready for 0.2.0-stub

### CODE REVIEW

File: Sources/CheckoutCardManagement/Constants.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ import Foundation
 
 enum Constants {
     /// Version of the SDK in use
-    static let productVersion = "0.1.1"
+    static let productVersion = "0.1.2"
```
Comment (alex-ioja-yang-cko) [inline Sources/CheckoutCardManagement/Constants.swift:12]:
```suggestion
    static let productVersion = "0.2.0"
```

File: version.txt [hunk 2]
```diff
@@ -1 +1 @@
-0.1.1
+0.1.2
```
Comment (alex-ioja-yang-cko) [inline version.txt:1]:
```suggestion
0.2.0
```

File: Sources/CheckoutCardManagement/LoggingSupport/LogEvent.swift [hunk 3]
```diff
@@ -30,6 +30,12 @@ enum LogEvent {
     /// Describe a successful call to retrieve a pan and a security code
     case getPanCVV(idLast4: String, cardState: CardState)
     
+    /// Describe a successfull event where a card state was completed
```
Comment (alex-ioja-yang-cko) [inline Sources/CheckoutCardManagement/LoggingSupport/LogEvent.swift:33]:
```suggestion
    /// Describe a successfull event where a card state change was completed
```

**Review by deepesh-vasthimal-cko — APPROVED** [review]
LGTM, Nice work.

**Review by fabio-insolia-cko — APPROVED** [review]

**Review by deepesh-vasthimal-cko — COMMENTED** [review]
LGTM.

**Pull request merged**